spec/javascripts/packs/form-steps-wait-spec.js

```diff
@@ -157,6 +159,23 @@ describe('FormStepsWait', () => {
       });
     });
 
+    context('invalid input', () => {
+
```

Comment:
Prettier formats this line away, which is one of the current build failures. Normally you can fix these with `yarn lint --fix` or use a plugin for your editor that formats it automatically on save.

```suggestion
```